File: Programmez/250/approval-testing/javascript/src/codeTestRunner.js
const TECHNICAL_WORKSHOP = require('./technicalWorkshop.js')

function runCodeTest(){
  TECHNICAL_WORKSHOP.addCat('Java')
  TECHNICAL_WORKSHOP.addCandidate('Toto', 'Titi', 'titi@mail.fr')

  const SCORE = TECHNICAL_WORKSHOP.run('Java')
  console.log(`The candidate as a total of ${SCORE} points.`)
}

module.exports = runCodeTest

File: Programmez/250/approval-testing/kotlin/src/main/kotlin/fr/codeworks/bbl/CodeTestRunner.kt
package fr.codeworks.bbl

import fr.codeworks.bbl.codeTest.TechnicalWorkshop

fun main() {
    val codeTest = TechnicalWorkshop()
    codeTest.addCat("SQL")
    codeTest.addCan("Toto", "Titi", "titi@mail.fr")
    val score = codeTest.runCodeTest("Java")
    println("The candidate as a total of $score points.")
}




